Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-buster
+
+RUN mkdir /autowebcompat
+WORKDIR /autowebcompat
+
+ADD Pipfile Pipfile.lock ./
+RUN pip install pipenv
+RUN pipenv install --system --deploy --ignore-pipfile
```

Comment:
Nit: you can install pipenv, run pipenv install, and then uninstall pipenv in the same line, so the resulting image will not have pipenv.

Reply:
Also use --no-cache-dir to avoid caching the package (not sure what's the equivalent for pipenv, if there is an equivalent).

You can also use `--disable-pip-version-check` to avoid useless logs.